src/model/schema.js

```diff
@@ -140,19 +148,6 @@ export default class Schema {
 		this._clearCache();
 	}
 
-	/**
-	 * Returns all registered items.
-	 *
-	 * @returns {Object.<String,module:engine/model/schema~SchemaCompiledItemDefinition>}
-	 */
-	getDefinitions() {
```

Comment:
Why this change? This method was here on purpose (next to `getDefinition()`).

Reply:
As funny as it might sound I have grabbed one method too much where moving `setAttributeProperties` and `getAttributeProperties`. Thanks for catching that.

Restored to the original position.